freqtrade/optimize/hyperopt.py

```diff
@@ -109,12 +109,12 @@ def __init__(self, config: Dict[str, Any]) -> None:
             if hasattr(self.custom_hyperopt, 'populate_indicators'):
                 self.backtesting.strategy.advise_indicators = (  # type: ignore
                     self.custom_hyperopt.populate_indicators)  # type: ignore
-            if hasattr(self.custom_hyperopt, 'populate_buy_trend'):
-                self.backtesting.strategy.advise_buy = (  # type: ignore
-                    self.custom_hyperopt.populate_buy_trend)  # type: ignore
-            if hasattr(self.custom_hyperopt, 'populate_sell_trend'):
-                self.backtesting.strategy.advise_sell = (  # type: ignore
-                    self.custom_hyperopt.populate_sell_trend)  # type: ignore
+            if hasattr(self.custom_hyperopt, 'populate_enter_trend'):
+                self.backtesting.strategy.advise_enter = (  # type: ignore
+                    self.custom_hyperopt.populate_enter_trend)  # type: ignore
+            if hasattr(self.custom_hyperopt, 'populate_exit_trend'):
+                self.backtesting.strategy.advise_exit = (  # type: ignore
+                    self.custom_hyperopt.populate_exit_trend)  # type: ignore
```

Comment:
You'll need to be very careful with this (especially with hyperopt).

Will require compat code.